### ISSUE TITLE
Tab sound icon always enabled in Safari

### ISSUE THREAD

**Issue by HiDeoo** [Adam13531/BotLand#250]
**Steps to reproduce**

1. Disable sounds

**Expected outcome**

Sounds disabled & tab sound icon disappearing.

**Actual outcome**

Sounds disabled & tab sound icon no disappearing.

This is probably due to:

```js
// Music needs to be set to 0 volume rather than stopping it.
```

Altho there is no explanation in the code on why, maybe in the notes somewhere.

**Browser information**

{"name":"Safari","safari":true,"version":"11.1","webkit":true,"mac":true,"a":true}

**In-game information**

Client version: 0.2.47
Player ID: 1
Player name: Adam

**Comment by freaktechnik** [comment]
As I'm sure is documented somewhere, the same also applies for Firefox.

**Comment by HiDeoo** [comment]
Finally found a todo from 8 months ago, this seems to be to avoid starting over the track when switching between page with music & page without music.

I'll check with Adam to be sure it's the only reason but if this it, there are multiple obvious workarounds 😄

**Comment by Adam13531** [comment]
Yes, I had only muted the music to avoid it from restarting. I think the music can simply be paused, that way unmuting it will cause it to resume where it left off rather than restart it again.
